corehq/apps/domain/forms.py

```diff
@@ -665,6 +683,26 @@ class PrivacySecurityForm(forms.Form):
         help_text=ugettext_lazy("Allow unknown users to request web access to the domain."),
     )
 
+    def __init__(self, *args, **kwargs):
+        super(PrivacySecurityForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper(self)
+        self.helper.form_class = 'form-horizontal'
+        self.helper.label_class = 'col-sm-3 col-md-2'
+        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
+        self.helper[0] = twbscrispy.PrependedText('restrict_superusers', '')
```

Comment:
@biyeun Without this hack boolean field is rendered in that way: 

![](https://sc-cdn.scaleengine.net/i/37aeb63b3b831ca782196aed386622a9.png)

I'm not sure maybe I'm doing something wrong but I've checked existing bootstrap3 form in hq and in my browser it also looks bad: (e.g. https://www.commcarehq.org/a/lukasz-test/indicators/import/). Maybe it's related to this problem: https://github.com/maraujop/django-crispy-forms/issues/225?